src/main/java/com/thebois/views/ColonyView.java

```diff
@@ -0,0 +1,42 @@
+package com.thebois.views;
+
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.glutils.ShapeRenderer;
+
+import com.thebois.models.beings.IBeing;
+import com.thebois.models.beings.IBeingGroup;
+
+/**
+ * Used to display all pawns.
+ */
+public class ColonyView implements IView {
+
+    private IBeingGroup colony;
+
+    /**
+     * Updates the views with current beings.
+     *
+     * @param beingGroup list of beings to draw.
+     */
+    public void update(IBeingGroup beingGroup) {
+        this.colony = beingGroup;
+    }
+
+    @Override
+    public void draw(final ShapeRenderer batch) {
+        float posX;
+        float posY;
+        final float radius = 5;
```

Comment:
Don't know how this looks nor how circle works in libGDX, but should we not use tileSize? Since it will occupy one tile?

And what did we decide on, should the pawn move freely from the grid (then we can make it a different size) or should they move by the grid (then they should be the size of one tile)?